pointers/pointers.c

```diff
@@ -8,7 +8,9 @@
 */
 void swap(int* a, int* b)
 {
-
+    *a += *b;
```

Comment:
you can use a temp variable to hold a, then assign b to a and b to temp. Another way of swapping